src/Util/RequestParser.php

```diff
@@ -32,7 +32,7 @@
      */
     public static function parseAndDuplicateRequest(Request $request) : Request
     {
-        $query = self::parseRequestParams($request->getQueryString());
+        $query = self::parseRequestParams($request->getQueryString() ?? '');
```

Comment:
`??` is the null coalesce operator. `?:` (shorthand version of ternary operator) works here too.